### PR TITLE
First bunch of error fixings.

### PR DESCRIPTION
**Pull request by scrdest**
Bunch o' error fixings. Porting urist guns to new system started but not finished. Didn't touch Vamp yet as per Steam message.

Commented out item_colors for time being, because I don't know if you will revert the Bay refactor or something.

### CODE REVIEW

**Comment by Glloyd** [comment]
You're a hero Scrd

File: code/game/verbs/who.dm [hunk 1]
```diff
@@ -169,4 +172,4 @@
 	if(config.show_mentors)
 		msg += "\n<b> Current Mentors ([num_mentors_online]):</b>\n" + mentmsg
 
-	src << msg
+	src << msg
```
Comment (scrdest) [inline code/game/verbs/who.dm:175]: The whole proc needs to be checked; Bay edited it and so did we way back when TGC got Coder, I didn't realize so I copied over Bay file altogether, then back from Urist master, so I'm not sure if it'll still work properly.

File: code/modules/urist/items/clothes/uristclothes.dm [hunk 2]
```diff
@@ -633,21 +633,21 @@ Update 26/07/2014 - All generic clothing goes under obj/item/clothing/under/uris
...
-	item_color = "relaxedwearm"
+	//item_color = "relaxedwearm"
 
 /obj/item/clothing/under/urist/enclaveo
 	name = "Enclave officer uniform"
 	desc = "It's a standard Enclave officer uniform.<br>The outer layer is made of a sturdy material designed to withstand the harsh conditions of the wasteland."
 	icon_state = "enclaveo"
 	item_state = "enclaveo"
-	item_color = "enclaveo"
+	//item_color = "enclaveo"
```
Comment (scrdest) [inline code/modules/urist/items/clothes/uristclothes.dm:650]: I realized just now all those item_colors were most likely entirely superfluous in the first place.

File: code/modules/urist/items/uristvanity.dm [hunk 3]
```diff
@@ -2,13 +2,14 @@
 
 Please keep it tidy, by which I mean put comments describing the item before the entry. -Glloyd*/
 
-//vanity lighters, stolen from the custom items.
+//vanity lighters, stolen from the custom items.//WHY. WHYYYYY -scrdest
 
-/obj/item/weapon/flame/lighter/zippo/vanity/blue
+/*/obj/item/weapon/flame/lighter/zippo/vanity/blue
```
Comment (scrdest) [inline code/modules/urist/items/uristvanity.dm:7]: Gotta find out if those variants still exist at all.

File: code/modules/urist/modules/uristjobs.dm [hunk 4]
```diff
@@ -35,7 +35,7 @@
 		H.equip_to_slot_or_del(new /obj/item/clothing/gloves/combat(H), slot_gloves)
 		H.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(H), slot_shoes)
 		H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/centcom(H), slot_w_uniform)
-		H.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/vest/fluff/deus_blueshield(H), slot_wear_suit)
+		H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/vest(H), slot_wear_suit) //regular-ass armor vest for now
```
Comment (scrdest) [inline code/modules/urist/modules/uristjobs.dm:38]: changed it to a generic for now, switch it to whatever you feel fits; I don't think we need a custom armor now that accessories are unlimited.

**Pull request merged**